l10n_it_asset_history_import_queue/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Cespiti: importazione storico dati in background
```

Comment:
***Italiano***

Questo modulo permette l'uso delle code per l'importazione dello storico dei cespiti.

***English***

This module adds job queue support to import assets history.